broadcast/broadcast-client-mock/mock_client_test.go

```diff
@@ -21,10 +21,10 @@ func TestMockClientSuccess(t *testing.T) {
 		expectedResult := []*broadcast.PolicyQuoteResponse{mocks.Policy1, mocks.Policy2}
 
 		// when
-		result, err := broadcaster.GetPolicyQuote(context.Background())
+		result, fail := broadcaster.GetPolicyQuote(context.Background())
 
 		// then
-		assert.NoError(t, err)
+		assert.Nil(t, fail)
```

Comment:
IMHO, we should change the returned type, to make those assert.NoError still working.

Reply:
it should be fixed - reverted NoError